linux/mozillavpn.spec

```diff
@@ -19,7 +19,7 @@ BuildRequires: golang >= 1.13
 BuildRequires: polkit-devel
 BuildRequires: python3-yaml
 BuildRequires: cargo
-BuildRequires: openssl
+BuildRequires: openssl-devel
```

Comment:
Ha, was this enough to fix the Fedora issue? :woman_facepalming:

Reply:
It was the `INTERFACE_LINK_LIBRARIES` property in `vpnglean/CMakeLists.txt` that really did the fixing.